### PR TITLE
feat: login, home, edgeResource, crd

### PR DESCRIPTION
**Pull request by githubcq**
/kind feature
feat: login, home, edgeResource, crd

### CODE REVIEW

**Comment by vincentgoat** [comment]
Thank you for your huge contribution.
Please squash the merge commit for cleanup, and sign all commits.

File: README.md [hunk 1]
```diff
@@ -5,6 +5,51 @@ KubeEdge dashboard provides a graphical user interface (GUI) for managing and mo
...
+or
+
+```bash with pnpm
+pnpm install
+```
+
+### Start project
+
+```bash with npm
+npm run dev --apiserver=[[proxy address, eg. https://192.168.33.129:6443]]
```
Comment (vincentgoat) [inline README.md:33]:
```suggestion
npm run dev --apiserver={proxy address}
Example: npm run dev --apiserver=https://192.168.33.129:6443
```

**Comment by vincentgoat** [comment]
image
The token field on the login page should be blank when it is initialized

File: README.md [hunk 2]
```diff
@@ -8,9 +8,48 @@ This project is currently in development, and we will iterate it continuously. W
...
+
+```bash with pnpm
+pnpm dev --apiserver=[[proxy address, eg. https://192.168.33.129:6443]]
+```
+
+### Login with token
+```bash
+kubectl create serviceaccount curl-user -n kube-system
+kubectl create clusterrolebinding curl-user-binding --clusterrole=cluster-admin --serviceaccount=kube-system:curl-user -n kube-system
+kubectl -n kube-system describe secret $(kubectl -n kube-system get secret | grep curl-user | awk '{print $1}'
```
Comment (vincentgoat) [inline README.md:54]:
```suggestion
kubectl -n kube-system describe secret $(kubectl -n kube-system get secret | grep curl-user | awk '{print $1}')
```

**Comment by vincentgoat** [comment]
image
Click namespace default on the home page and showed not found error.

**Comment by vincentgoat** [comment]
image
1. Update deployment failed with error `code: 409, message: "Operation cannot be fulfilled on deployments.apps \"test123\": the object has been modified; please apply your changes to the latest version and try again"`

2. Although the update operation failed, the contents of the yaml were updated, but not in K8s.

**Comment by vincentgoat** [comment]
image
Can't find a way to check the details of configmap

**Comment by githubcq** [comment]
> image The token field on the login page should be blank when it is initialized

It is fixed.

**Comment by githubcq** [comment]
> image Click namespace default on the home page and showed not found error.

It is fixed.

**Comment by githubcq** [comment]
> image
> 
> 1. Update deployment failed with error `code: 409, message: "Operation cannot be fulfilled on deployments.apps \"test123\": the object has been modified; please apply your changes to the latest version and try again"`
> 2. Although the update operation failed, the contents of the yaml were updated, but not in K8s.

For problem 1, I tried to modify replicas number in yaml successfully. The `code: 409` results from wrong modified param, which is unable to pass k8s verification.
For problem 2, it is fixed.

File: README.md [hunk 3]
```diff
@@ -8,9 +8,51 @@ This project is currently in development, and we will iterate it continuously. W
...
+## Install packages
+
+```bash with npm
+npm install
+```
 
 ## Contributing
 If you're interested in being a contributor and want to get involved in developing the KubeEdge code, please see [CONTRIBUTING](./CONTRIBUTING.md) for details on submitting patches and the contribution workflow.
 
 ## License
```
Comment (vincentgoat) [inline README.md:22]: Please move the sections `Contributing` and `License` to the end of this md.

**Comment by vincentgoat** [comment]
image
Error message pops up on the login page.

**Comment by githubcq** [comment]
> image Error message pops up on the login page.

It is fixed.

**Pull request merged**